lambda_data_logger/main.py

```diff
@@ -117,6 +119,45 @@ def handler(event, context):
 
     logger.info(f"Uploaded updated {object_name} to S3")
 
+    # GitHub Teams with CoPilot Data
+
+    logger.info("Getting GitHub Teams with CoPilot Data")
+
+    copilot_teams = []
+
+    response = gh.get(f"/orgs/{org}/teams", params={"per_page": 100})
+
+    # Get the last page of teams
+    try:
+        last_page = int(response.links["last"]["url"].split("=")[-1])
+    except KeyError:
+        last_page = 1
+
+    for page in range(1, last_page + 1):
+        response = gh.get(f"/orgs/{org}/teams", params={"per_page": 100, "page": page})
+        teams = response.json()
+        for team in teams:
+            usage_data = gh.get(f"/orgs/{org}/team/{team['name']}/copilot/usage")
+            try:
+                if usage_data.json() != []:
+                    copilot_teams.append(team["name"])
+            except Exception as error:
+                # If Exception, then the team does not have copilot usage data and can be skipped
+                pass
```

Comment:
Could this be split into a helper function instead of having a nested loop?

Also the try/except should have a comment explaining that the `usage_data` process will definitely return nothing if. I didn't know that whether it could possibly return something in JSON but not with usage data.

Both these comments are tiny and don't affect runtime, just thought I'd add.

Reply:
Done